src/odb/src/db/dbTechLayer.cpp

```diff
@@ -740,6 +752,7 @@ dbIStream& operator>>(dbIStream& stream, _dbTechLayer& obj)
   stream >> *obj.width_table_rules_tbl_;
   stream >> *obj.min_cuts_rules_tbl_;
   stream >> *obj.area_rules_tbl_;
+  stream >> *obj.forbidden_spacing_rules_tbl_;
```

Comment:
You need to make this conditional of the schema - see db_schema_keepout_zone below

Reply:
Thanks! That fixed the read_db unit test.